Test.C

```diff
@@ -128,20 +128,22 @@ int main(){
     player.currentCredits = 30;
 
     while(player has options and more rounds exists){
-
-        //output player status (if statements)
-        //current credit
-        //current equipment
-        //pulls from info from index amount table referencing index struct table
-                /* WHY DO WE NEED TO USE MULTIPLE ARRAYS FOR EQUIPMENT INSTEAD OF A LIST WITH A QUANTITY ATTRIBUTE? */
-        //current encrypt
-        //iterate and print through current encryption
+        //receive credits NATHAN
+        if(end of round){ //output player status (if statements)
+            player.currentCredits = player.currentCredits - (equipment.cost + encryption.cost); //current credit
```

Comment:
This line does not make sense here. The purpose of the output player status section is to use a whole bunch of `printf` statements to show the player what they currently have. This is the presentation part of the code, so no values should be changing. The challenge for this section is to present everything neatly and concisely.
```suggestion
            printf("\nCredits: %d", player.currentCredits);
```